src/components/TransferOrderItem.vue

```diff
@@ -100,6 +100,12 @@ export default defineComponent({
     }),
   },
   methods: {
+    setProgressBarColor(event:any, item: any) {
+         const fraction = this.getPickedToOrderedFraction(item);
```

Comment:
Improve indentation, we follow indentation of two spaces across app.

Reply:
I have improved the indentation.